engine/jobmaster/dm/metadata/job.go

```diff
@@ -120,15 +121,17 @@ func NewJob(jobCfg *config.JobCfg) *Job {
 // Task is the minimum working unit of a job.
 // A job may contain multiple upstream and it will be converted into multiple tasks.
 type Task struct {
-	Cfg   *config.TaskCfg
-	Stage TaskStage
+	Cfg         *config.TaskCfg
+	Stage       TaskStage
+	CreatedTime time.Time
 }
 
 // NewTask creates a new Task instance
 func NewTask(taskCfg *config.TaskCfg) *Task {
 	return &Task{
-		Cfg:   taskCfg,
-		Stage: StageRunning, // TODO: support set stage when create task.
+		Cfg:         taskCfg,
+		Stage:       StageRunning, // TODO: support set stage when create task.
+		CreatedTime: time.Now(),
```

Comment:
So the CreatedTime means the Job created time, not unit created time?

Reply:
I mixed the Job and Worker in engine before. After the discussion with lance, I decide to add `createdTime` inside `CreateWorker`. What do you think about?

Reply:
Will the value lost when failover?

Reply:
Store it in the `JobStore` can't be lost